vnext/docs/Testing.md

```diff
@@ -0,0 +1,25 @@
+# Testing ReactNative for Windows
+
+When contributing to this project, unit and integration tests should be run to help prevent new bugs and regressions.
+
+## Windows Desktop
+Tests should be run with a VSTest-compatible client
+(i.e. [VSTest.Console.exe](https://docs.microsoft.com/en-us/visualstudio/test/vstest-console-options?view=vs-2019),
+[Visual Studio Test task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops)).<br/>
+This project includes convenience scripts to set up and run the test artifacts.
+
+### Unit Tests
+*Implemented in the React.Windows.Desktop.UnitTests project.*<br/>
+Isolated, self-contained tests covering concrete implementations against mock dependencies and data.
+```powershell
+& Scripts\UnitTests.ps1
```

Comment:
Is that the actual command:
& Scripts\UnitTests.ps1

That's how I see it rendered in VsCode

Reply:
That seems correct to me. Is that a problem?

Reply:
Why do we need "&" in front of the line? I'm just not as much familiar with PowerShell

---
In reply to: [304699399](https://github.com/microsoft/react-native-windows/pull/2768#discussion_r304699399) [](ancestors = 304699399)

Reply:
'&' is PowerShell's "execute". However, I think it's not needed here.

Reply:
Not needed, doesn't hurt.
It can be either:
- .\Scripts\UnitTests.ps1
- & "$(some+path)\Scripts\UnitTests.ps1"
- & .\Scripts\UnitTests.ps1